src/directives/colony/outpostSK.ts

```diff
@@ -12,21 +15,74 @@ export class DirectiveSKOutpost extends Directive {
 	static directiveName = 'outpostSK';
 	static color = COLOR_PURPLE;
 	static secondaryColor = COLOR_YELLOW;
+	static containerRepairTheshold = 0.5;
+
+	memory: DirectiveSKOutpostMemory;
 
 	constructor(flag: Flag) {
 		super(flag, colony => colony.level >= 7);
 	}
+	
+	isCenterRoom(): boolean {
+		const coords = Cartographer.getRoomCoordinates(this.pos.roomName);
+		if(coords.x % 10 != 0 && coords.x % 5 == 0 && coords.y % 10 != 0 && coords.y % 5== 0){
+			return true;
+		}
+		return false;
+	}
 
 	spawnMoarOverlords() {
-		this.overlords.sourceReaper = new SourceReaperOverlord(this);
+		if(this.memory.isCenterRoom == undefined){
+			this.memory.isCenterRoom = this.isCenterRoom();
```

Comment:
don't store `isCenterRoom` in memory. it's inexpensive to calculate and can be stuck directly in the DirectiveSKOutpost constructor

Reply:
done, it is now checked in constructor.
existing Overmind users that already have the directive flag in place will not benefit from this, would you suggest to add it under init() too?